spring-boot-project/spring-boot-starters/spring-boot-starter-pulsar/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+	id "org.springframework.boot.starter"
+}
+
+description = "Starter for using Spring for Apache Pulsar"
+
+dependencies {
+	api(project(":spring-boot-project:spring-boot-starters:spring-boot-starter"))
+	api("com.github.ben-manes.caffeine:caffeine")
```

Comment:
There is WIP to shade the fact that the cache mechanism is using caffeine. It is an internal detail and not intended to swapped out.

Reply:
This is done now.